src/BulkAppointments/OffenderUpload/OffenderUpload.styles.js

```diff
@@ -3,13 +3,12 @@ import { SPACING } from '@govuk-react/constants'
 
 // eslint-disable-next-line import/prefer-default-export
 export const Container = styled('form')`
-  margin-top: ${SPACING.SCALE_5};
+  margin-bottom: ${SPACING.SCALE_5};
   display: flex;
   flex-direction: column;
-  justify-content: space-around;
+  // justify-content: space-around;
```

Comment:
can this be removed rather than commented out?

Reply:
🤢 good spot!